app/assets/javascripts/users.js

```diff
@@ -0,0 +1,25 @@
+function getRepositories() {
+  var url = $("#repositories").data("repositories")
```

Comment:
Maybe you can pass the URL in other way instead of fetching it straight from the DOM